src/deluge/model/instrument/midi_instrument.h

```diff
@@ -73,7 +73,8 @@ class MIDIInstrument final : public NonAudioInstrument {
 
 	int8_t modKnobCCAssignments[kNumModButtons * kNumPhysicalModKnobs];
 
-	MPEOutputMemberChannel mpeOutputMemberChannels[15]; // Numbers 1 to 14. 0 is bogus
```

Comment:
The comment that was there before is bugging me ( // Numbers 1 to 14, 0 is bogus).
Does that mean that the code in .cpp file should be checking always from index 1 of the array and ignoring mpeOutputMemberChannels[0] ? it should ignore it for notes if MPE Lower Zone but should be taken into account if MPE UPPER ZONE. All code in the cpp file is ignoring the last member channel of either zone, first because they set the length of this array to 15 so last channel was always impossible to select, and by adding that comment they are trying to tell us not to use index 0 of the array, which we can see the like in line 52:

```
    for (int32_t c = 1; c <= 14; c++) {
		mpeOutputMemberChannels[c].lastNoteCode = 32767;
		mpeOutputMemberChannels[c].noteOffOrder = 0;
	}
```

Reply:
I think we are doing it right now with your modifications because we take into account channel 15 for lower zone, and channel 0 for upper zone.

Reply:
Ya, it's correct in official only because of a different bug which causes channel 1 and 16 to always be treated as MPE